src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,283 @@
+---
+part_number: I
+chapter_number: 3
+title: Markup
+description: Markup chapter of the 2019 Web Almanac covering elements used, custom elements, value, products, and common use cases.
+authors: [bkardell]
+reviewers: [zcorpan, tomhodgins, matthewp]
+translators: [SilentJMA]
+discuss: 1758
+published: 2019-11-27T00:00:00.000Z
+last_updated: 2019-11-27T00:00:00.000Z
+---
+
+## Introduction
+
+En 2005, Ian "Hixie" Hickson a publié [une analyse des données de balisage](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html) à partir de différents travaux précédents. Une grande partie de ce travail visait à examiner les noms de classes pour voir s’il existait une sémantique informelle commune adoptée par les développeurs et sur laquelle il serait peut-être logique de normaliser. Certaines de ses recherches ont permis d’informer de nouveaux éléments en HTML5.
+
+14 ans plus tard, il est temps de jeter un nouveau coup d’œil. Depuis ce temps, nous avons introduit [des éléments personnalisés](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) et l'[Extensible Web manifesto](https://extensiblewebmanifesto.org/), ce qui nous a encouragé à trouver des meilleures méthodes pour tracer des chemins en permettant aux développeurs d'explorer l'espace des éléments eux-mêmes et permettent aux organismes de normalisation [d'agir comme des éditeurs de dictionnaire](https://bkardell.com/blog/Dropping-The-F-Bomb-On-Standards.html). Par contre les noms de classes CSS, qui peuvent être utilisés pour n'importe quoi, nous pouvons être certains que les auteurs qui ont utilisé un élément {en3} ne sont pas au standard {/en3} ont vraiment voulu qu'il soit un élément.
```

Comment:
yes i really got perplexed on it / si on trace un chemin ça serai un nouveau / but if it was an existing road on peut utulisé "baliser " et aussi "paver" because its already existed so on va l'ameliorer ??